lib/cc/engine/analyzers/reporter.rb

```diff
@@ -38,9 +38,11 @@ def report
           flay.report(StringIO.new).each do |issue|
             violation = new_violation(issue)
 
-            unless reports.include?(violation.report_name)
-              reports.add(violation.report_name)
-              io.puts "#{violation.format.to_json}\0"
+            violation.occurrences.each do |occurrence|
```

Comment:
This is the other big change, to me. I have the same thinking about this as about the mass calculation change.